lib/client.js

```diff
@@ -157,6 +160,8 @@ class Client extends Dispatcher {
     this[kSocket] = null
     this[kPipelining] = pipelining != null ? pipelining : 1
     this[kMaxHeadersSize] = maxHeaderSize || 16384
+    this[kBodySize] = 0
+    this[kResponseContentLength] = 0
```

Comment:
make sure these are reset at the beginning of dispatch.

Reply:
These can live on the parser instance and don’t need symbols.

Reply:
I moved the vars to `Parser`, is the reset still needed?